examples/remarketing/add_remarketing_action.py

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/env python
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This example demonstrates usage of remarketing actions.
+
+A new remarketing action will be created for the specified customer, and its
+associated tag snippets will be retrieved.
+"""
+
+
+import argparse
+import sys
+import uuid
+
+import google.ads.google_ads.client
+
+
+def main(client, customer_id, page_size):
+    remarketing_action_resource_name = _add_remarketing_action(
+        client, customer_id)
+
+    print(f'Created remarketing action "{remarketing_action_resource_name}".')
+
+    queried_remarketing_action = _query_remarketing_action(
+        client, customer_id, remarketing_action_resource_name, page_size)
+
+    _print_remarketing_action_attributes(client, queried_remarketing_action)
+
+
+def _add_remarketing_action(client, customer_id):
+    remarketing_action_service = client.get_service(
+        'RemarketingActionService', version='v2')
+
+    # Create the operation.
```

Comment:
I thought we are trying to get away from comments that state the obvious. Either this and the next comment should be removed, which is preferable, or another another trivial comment to the line above for consistency.